dali/kernels/signal/fft/fft_cpu_impl_ffts.cc

```diff
@@ -77,8 +77,8 @@ KernelRequirements Fft1DImplFfts<OutputType, InputType, Dims>::Setup(
   auto out_shape = in.shape;
 
   ScratchpadEstimator se;
-  se.add<mm::memory_kind::host, float>(size_in_buf(nfft),  32);
-  se.add<mm::memory_kind::host, float>(size_out_buf(nfft), 32);
+  se.add<mm::memory_kind::pinned, float>(size_in_buf(nfft),  32);
+  se.add<mm::memory_kind::pinned, float>(size_out_buf(nfft), 32);
```

Comment:
Why pinned here? Is it used beyond ffts?

Reply:
Copy-find-replace backfired. I'll revert.